src/cli/providers/messenger/persona.js

```diff
@@ -46,13 +47,15 @@ const help = () => {
 };
 
 export async function createPersona(ctx) {
-  const { t, token, name: personaName, pic: personaUrl } = ctx.argv;
+  const token = ctx.argv['--token'];
+  const personaName = ctx.argv['--name'];
+  const personaUrl = ctx.argv['--pic'];
```

Comment:
@kidkcl Do you have any idea about this naming?
I just refactor the origin codebase but feel weird about the naming.

Reply:
Agree. This should be renamed. #341